include/swift/AST/Decl.h

```diff
@@ -4592,6 +4592,23 @@ class AbstractStorageDecl : public ValueDecl {
   /// with caution.
   AccessorDecl *getSynthesizedAccessor(AccessorKind kind) const;
 
+  /// Return an accessor part of the set of opaque accessors dictated by the
+  /// requirements of the ABI.
```

Comment:
"Return an accessor *that's known to be* part of the set..."

Technically, it's possible to distinguish between *opaque* accessors, which are guaranteed-to-exist ABI-stable implementations of the core access operations, and *accessors that might be part of the (non-stable or frozen) ABI*.  For example, we might want to do funny things with `didSet` in the future.  Not sure if this is worth spelling out in the comment.